src/openseadragon.js

```diff
@@ -230,6 +230,9 @@
   * @property {Boolean} [flipped=false]
   *     Initial flip state.
   *
+  * @property {Boolean} [overlayContentFlipped=false]
```

Comment:
We need a good name for this, which I think is going to be a challenge. It needs to communicate "if the viewport is flipped, flip the contents of overlays back so they still read correctly" but in fewer words. Maybe something like `overlayFlipCompensation`.

Then the description below should be much more detailed to explain the purpose of this flag.

Reply:
How about `overlayFlipReversal` or `overalScaleReversal`?